scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/spi/model/ModelValidatorContext.java

```diff
@@ -261,5 +334,41 @@ public Boolean getValid() {
         public void setValid(Boolean valid) {
             this.valid = valid;
         }
+
+        public boolean hideVar(String variableName) {
```

Comment:
I feel a bit confused about these two funcs. They are used to change the variable state stored in the map, and always return true if it's a hidden variable (i.e., no matter what's its current state)? Could you add a comment?

Reply:
it always returns true because it does not perform a validation. It performs an action. Returning true allows to use these functions easily in a spel expression. e.g. showVar('var1) && hideVar('var2')